test/unit/model/virtualbox-test.js

```diff
@@ -240,10 +240,23 @@ describe('Virtualbox installer', function() {
       }
     });
 
+    it('should run virtualbox installer executable to extract msi installer into temp folder', function(done) {
+      let stub = sandbox.stub(child_process, 'execFile').yields(null, "Files were extracted to temp", "");
+      try {
+        installer.install(fakeProgress, success, failure);
+        done();
+      } catch (error) {
+        expect.fail();
+      }
+      return helper.execFile().then(() => {
+        expect(stub).to.be.calledOnce;
+        expect(stub).to.be.true;
```

Comment:
this looks wrong, you instead should be checking that stubbed method called with right parameters. In your case you should check that downloaded executable file is called to unpack to temp folder.